_extras/checkout.md

```diff
@@ -247,10 +247,11 @@ also email you to let you know, as this information is typically not shared duri
 after you have attended your teaching demonstration.
 
 ## 4. Tracking Your Progress
-After you complete this workshop, your application will be matched to your training record and a profile will be created in The
+After you complete this workshop, your application or profile-creation form will be matched to your training record and used to 
+create a profile that you can access in The
```

Comment:
```suggestion
<X time> after your instructor training workshop workshop, Carpentries Core Team members will activate
your profile in The
```
maybe simplify it to just what impacts the trainee instead of what happens a little bit?  

Do trainees get notified by AMY when they can access it?